cmd/server/flags.go

```diff
@@ -97,6 +97,12 @@ var flags = []cli.Flag{
 		Name:    "authenticate-public-repos",
 		Usage:   "Always use authentication to clone repositories even if they are public. Needed if the SCM requires to always authenticate as used by many companies.",
 	},
+	&cli.StringFlag{
```

Comment:
cli.StringSliceFlag ?

Reply:
Should be a string. Why use string slice?